Assets/Character/AaronTestAssets/PlayerState.cs

```diff
@@ -0,0 +1,74 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerState : MonoBehaviour {
+
+
+    private Vector3 scale;
+    protected float direction = 1;
+    protected int facing = 1;
```

Comment:
Doesn't this mean that each state has its own idea of what direction it's facing? I feel that this should be known by all of them. Also, you might want to document what these numbers mean, what is left, what is right etc.